content/en/account_management/saml/troubleshooting.md

```diff
@@ -0,0 +1,147 @@
+
+### No active account for a user
+
+Enabling JIT provisioning may result in the `There is no active account for` error if a user has not received an email invitation or if their account is disabled in [User settings][3]. To resolve, follow the steps below based on your scenario:
+
+  - If a new Datadog user tries to log into a Datadog org without an email invitation and the `There is no active account for error` occurs, send the user an email invitation and check if JIT provisioning is enabled for your org. The user **must** accept the invitation to enable JIT provisioning. Once the invitation is sent, the user exists in the system and cannot use JIT provisioning until the invite is accepted. If a user needs to verify their email or the invitation is expired, have the admin send a new invitation.
+  - If a user is no longer enabled in a Datadog organization that has JIT provisioning enabled and they try to login again through SAML and the `There is no active account for error` occurs, re-enable the user in [User settings][3].
+
+## IdP metadata file errors
+
+If you are having trouble updating your IdP metadata file, verify that the metadata file you are trying to upload is valid.
+
+To validate your metadata file:
+
+1. Choose a SAML validation tool, such as the [SAML developer tool][4] by OneLogin.
+2. Paste your metadata into the XML field and select **Metadata** in the XSD (schema file) field.
+3. Click **Validate XML With the XSD Schema**.
+
+## Roles errors
+
+When mappings are enabled, users logging in with SAML to a Datadog account are stripped of their current roles and reassigned to new roles based on the details in their SAML assertion passed on from your Identity Provider.
+
+Users who log in with SAML and do not have the values that map to a Datadog role are stripped of all roles and are not allowed to log in.
+
+{{< img src="account_management/saml/unknown_user_error.png" alt="No AuthNMappings for this user" style="width:80%;">}}
+
+If you have group mappings set and are not able to see your roles, your group mappings in the Datadog application may appear differently in your IdP. To verify:
+
+1. Retrieve your IdP's SAML assertion for your account. Use browser tooling, such as [extensions][5], to retrieve your SAML assertion. For example:
+
+  ```xml
+  <saml2:Attribute Name="member_of"
+                             NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+                             >
+                <saml2:AttributeValue xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                                      xsi:type="xs:string"
+                                      >name_of_your_group_goes_here</saml2:AttributeValue>
+  </saml2:Attribute>
+  ```
+
+2. Navigate to the Team page in the bottom left corner of Datadog.
```

Comment:
The "Team" page no longer exists! These instructions need updating and verification.